### ISSUE TITLE
Thread migration, group renaming, group deletion, etc.

### ISSUE THREAD

**Issue by aureooms** [edemaine/coauthor#208]
Those actions cannot be done in the UI and I understand that must be because they are destructive in nature (what happens to permalinks for example?). It should at least be documented how to do this via the mongodb repl as it is something someone certainly would like to be able to do(?), if it is doable in the first place.

**Comment by edemaine** [comment]
I agree.  What do you mean by "thread migration"?  Between groups?

**Comment by aureooms** [comment]
Yes, a thread migration would be moving a thread from one group to another.

A related question is: "Why not per thread permissions?" I guess you must already have discussed that with others. To me, coauthor would be a replacement for email communications, with better search, formatting and sharing options, and removing the distraction of unrelated stuff that usually pollutes mailboxes. A thread on coauthor would be the equivalent of an email thread. Threads would be private to a set of users, could be "forwarded" by adding read and write permissions to an additional user. For example, currently, a user must have access to all threads of a group if he just wants to read one of them.

In short, groups are great for workshops but not so much for more private work where loose threads would make more sense. What do you think about that? Maybe it is not the aim of your project.

**Comment by edemaine** [comment]
Per-thread permissions is #79.  It's my top priority at the moment.

**Comment by aureooms** [comment]
Cool!

**Comment by edemaine** [comment]
Group renaming should now be working (though it's still a dangerous operation... could break a lot of links).
